includes/class-cli.php

```
@@ -82,6 +82,67 @@ public function meta( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Refresh Webmentions by post or by comment_id
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--comment=<comment_id>]
+	 * : The comment ID
+	 *
+	 * [--post=<post>]
+	 * : Instead of just a comment ID, support using a post ID or post URL
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp webmention refresh 1
+	 *
+	 *     $ wp webmention refresh --post=http://example.com/post/1
+	 *
+	 *     $ wp webmention refresh --post=1
+	 *
+	 * @param array|null $args       The arguments.
+	 * @param array|null $assoc_args The associative arguments.
+	 *
+	 * @return void
+	 */
+	public function refresh( $args, $assoc_args ) {
+		if ( $assoc_args ) {
```

I would rewrite this code to fail fast.

```PHP
if ( !$assoc_args ) {
    WP_CLI::error( __( 'Please provide a comment ID or a post-id/permalink', 'webmention' ) );
}